### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#26] [General]
We should decide what documentation should be produced prior to development commencing, during development and post implementation.

A high level overview of best practices for agile documentation can be found below however I'd welcome people's thoughts and own experiences.

https://easternpeak.com/blog/agile-documentation/

**Comment by harperjohn** [comment]
I am really like Hugo for documentation  https://gohugo.io/

There are lots of existing Documentation themes https://themes.gohugo.io/tags/documentation/

Docsy has caught my eye https://themes.gohugo.io/docsy/

Like this one too https://docs.neuralvibes.com/  (I like that it has "dark mode")

**Comment by DotNetDublin** [comment]
Thanks John. I've heard the cool kids mentioning Hugo on various podcasts so would be happy to explore that option.

As well as the documentation tool we need to decide what documentation we are going to produce and when.

**Comment by ripteqdavid** [comment]
Another nice documentation system I've been playing with is GitBooks https://www.gitbook.com/

**Reply by DotNetDublin** [reply]
That looks really good and I see it has a free plan for open source projects.
